UKHomeOffice/cerberus-service
# [COP-10952, COP-10746 ] Add tests for checking highest threat level with Selectors & rules

## Description
 Add tests for checking highest threat level with Selectors & rules
Add test to check rule with name "Selector Matched Rule" is not displayed.
"When there is a selector match, then a rule match is also generated. 
This rule match should not be displayed or included in any rule counts"

## To Test
npm run cypress:runner

run the following tests
`Should verify Matched Rules with rule name Selector Matched Rule not visible on task details page`
`Should verify Matched Rules with rule name Selector Matched Rule not visible on task details page`
`Should check rule matches details on task details page`
`Should check  rule matches details for more than one version on task management page`
## Developer Checklist

\* Required
